.github/workflows/run_black_linter.yml

```diff
@@ -4,11 +4,25 @@ on:
   push:
     branches:
       - main
+    paths:
+      - 'backend/**'
```

Comment:
Would it not be better to just check if `*.py` was changed, since black will run on all python files

Reply:
fixed